### ISSUE TITLE
使用w2ner训练时报错

### ISSUE THREAD

**Issue by yuxin233** [zjunlp/DeepKE#352]
我在使用w2ner时，出现这个错误

![1698151824182](https://github.com/zjunlp/DeepKE/assets/109857514/540be336-0fc1-4b4d-adb5-b67a42db5398)

我的train.yaml:
data_dir: "../data"
save_path: "output"
device: 2 # cuda

do_train: True
do_eval: False
do_predict: True

epochs: 15           # the number of training epochs
batch_size: 8

warm_factor: 0.1  # warmup rate
weight_decay: 0.1
clip_grad_norm: 5.0
bert_learning_rate: 1e-5 # lr for bert parameters
learning_rate: 1e-3  # lr for other parameters

max_seq_len: 200 # Max Word length (Bert: 510 ([CLS], [SEP]  ----->  512), Under 500 is recommended)
use_bert_last_4_layers: True
seed: 123

**Comment by pengzju** [comment]
- 从报错信息来看，是加载模型产生了错误。请您检查`conf/model.yaml`文件中的`bert_name`字段是否正确。
- 另外您可以选择将此处的权重下载到本地(https://huggingface.co/bert-base-chinese)，并且将`bert_name`字段设置为本地路径。

**Comment by yuxin233** [comment]
您好，改变模型路径后出现这个错误
![image](https://github.com/zjunlp/DeepKE/assets/109857514/721ec796-32f0-4233-8acc-615eb15a4449)
然后我手动在config.json里面添加了 “model_type" ="bert", 然后出现了这个错误：
![image](https://github.com/zjunlp/DeepKE/assets/109857514/8097e397-0d19-43fc-a361-a9e75e4baac0)

**Comment by pengzju** [comment]
- 1. 您的模型是在哪里下载的？具体的网址可以发我一下吗？
- 2. 看起来是您的GPU型号不匹配，这个问题您可以更新Nvidia驱动版本，使其与cuda版本相匹配

**Comment by yuxin233** [comment]
1. 模型下载地址 https://drive.google.com/file/d/1AQitrjbvCWc51SYiLN-cJq4e0WiNN4KY/view，用的是deepke/example/ner/standard 要求的bert模型，看来要换成您说的模型
2. 谢谢您，我明白了
